core/settings.py

```diff
@@ -64,8 +64,10 @@
     "django.contrib.staticfiles",
     "django_extensions",
 
-    'home',
-    'theme_pixel'
+     'home',
+    'theme_pixel',
+    'rest_framework',  
```

Comment:
Please add djangorestframework and its version in requirements.txt file

Reply:
![image](https://github.com/user-attachments/assets/bc90ac98-4961-4865-a2eb-82d3fcf649f4)